Package.resolved

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
Tohle by tu nemělo být.

Reply:
Nemělo by to tu být jenom protože to není aplikace, ale knihovna. Nicméně takto je aspoň jistota, že na CI a u nás bude stejná verze závislostí. Stejný princip a záležitost kvůli které máme v repozitářích `Gemfile.lock` a `Podfile.lock`.

Reply:
Já jsem se na to právě taky koukal, píšou to v tom evolution proposal. Ještě jsem checknul že to tam nenáme tak jsem to postnul.

Reply:
Sorry, jsem zapomněl že to musím sám approvnout.